scripts/gui.lua

```diff
@@ -1428,6 +1575,9 @@ function GUI.createTrackerConfigAndInfo(trackerInfo, tracker)
     GUI.updateTrackerInfo(trackerInfo, tracker)
 end
 
+---comment
```

Comment:
interesting comment :stuck_out_tongue:

Reply:
If the function name is self-explanatory, you can leave out the function description (at least I do :wink:)